lib/salus/repo.rb

```diff
@@ -55,7 +55,7 @@ class Repo
     IMPORTANT_FILES.each do |file|
       define_method :"#{file[:handle]}_present?" do
         if file[:wildcard]
-          files = Dir["#{@path_to_repo}/**/*#{file[:filename]}"]
+          files = run_rg("rg", "--files", "-g", file[:filename])
```

Comment:
What do `--files` and `-g` mean? could add a comment in the code explaining these?

Reply:
Maybe even add a brief description of `rg` with its url.